docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/rancher-prime-aws.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
I started leaving comments, but there are a lot of line breaks in the middle of sentences in this file, which makes it hard to make edit suggestions. Could you clean up the line breaks?